 Tue, 03 Mar 2020 22:57:19  INFO logger.py <module> [ line:20 ] hehehehe
 Tue, 03 Mar 2020 22:57:19  ERROR logger.py <module> [ line:21 ] heheheheheheh
Traceback (most recent call last):
  File "D:\Program Files (x86)\python\lib\site-packages\_pytest\config\__init__.py", line 456, in _importconftest
    return self._conftestpath2mod[key]
KeyError: WindowsPath('D:/python/mystudy/AppFramework/TestCases/conftest.py')
 Tue, 03 Mar 2020 22:57:19  INFO conftest.py init_session [ line:65 ] xxxxxxx
 Tue, 03 Mar 2020 22:58:17  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_login')查找成功，耗时0.5231606960296631s
 Tue, 03 Mar 2020 22:58:17  INFO test_0_login.py test_prepare [ line:17 ] app已登录,执行数据清空操作!
